### ISSUE TITLE
add support for transip managed kubernetes

### ISSUE THREAD

**Issue by KiaraGrouwstra** [aequitas/terraform-provider-transip#72]
see https://www.transip.nl/kubernetes/

**Comment by nicwortel** [comment]
I would also be interested in this.

The relevant API documentation can be found here: https://api.transip.nl/rest/docs.html#kubernetes

**Comment by NielsCodes** [comment]
Yes please! We're looking to move workloads away from the American cloud providers, but Terraform support is must-have. I guess we'll have to skip TransIP for the time being.

**Comment by aequitas** [comment]
Implementing this will take a considerable amount of time in testing I'm afraid. But I currently have no way of testing this as I don't have a Kubernetes cluster on Transip and I don't have much spare time at the moment to dedicate to this project.
